apps/frontend/src/hooks/useAssetBalance.ts

```diff
@@ -34,8 +37,9 @@ export const useAssetBalance = (
     [address, walletIndex, wallets],
   );
 
-  const [state, setState] = useState<CacheCallResponse<string>>({
+  const [state, setState] = useState<AssetBalanceResponse>({
     value: '0',
+    parsedValue: '0',
```

Comment:
It's a bit weird naming because `value` is basically `weiBalance` and `parsedValue` is `balance`. I'd propose to rename them so it's more clear.